src/upload.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+const CHUNK_SIZE = 64 * 1024 * 1024; // 64MB
+
+export default function upload({ url, file, videoId }) {
+  const { size } = file;
+  if (CHUNK_SIZE > file.size) return post({ url, videoId, chunk: file });
+  const chunks = createChunks(file);
+  return Promise.all(chunks.map(it => post({ url, videoId, size, ...it })));
```

Comment:
Does it make sense to handle the response here and provide a single return value to further handle the outcome where this function is called?